R/generate_event_table_1.R

```diff
@@ -0,0 +1,74 @@
+#' Generate table of event counts from user input, including two conditions: common control and overlap population
+#'
+#'
+#' @param input_data user input of population at IA and FA
+#' @param hypothesis Illustrate groups want to compare
+#' @param type either common control or overlap population
```

Comment:
Please update to @param type Either `"common_control"` or `"overlap_population"`